includes/activity/class-activity.php

```diff
@@ -156,11 +158,11 @@ class Activity extends Base_Object {
 	 * Set the object and copy Object properties to the Activity.
 	 *
 	 * Any to, bto, cc, bcc, and audience properties specified on the object
-	 * MUST be copied over to the new Create activity by the server.
+	 * MUST be copied over to the new "Create" activity by the server.
 	 *
 	 * @see https://www.w3.org/TR/activitypub/#object-without-create
 	 *
-	 * @param array|string|Base_Object|Link|null $data Activity object.
+	 * @param array|string|Activity|Link|null $data Activity object.
```

Comment:
It can be both... but a `Base_Object` is more common!

Reply:
With `Base_Object` I get warnings for potentially polymorphic calls within the function. Maybe the better way would be to document these magic methods in `Base_Object` (as well)?

Reply:
what IDE plugin do you use for that?

Reply:
no plugin, just phpstorm itself